frontend/public/components/notification-drawer.tsx

```diff
@@ -108,20 +111,37 @@ const getUpdateNotificationEntries = (
   isLoaded: boolean,
   updateData: ClusterUpdate[],
   isEditable: boolean,
+  newerChannel: string,
+  newerChannelVersion: string,
```

Comment:
Instead of passing `updateData`, `newerChannel`, and `newerChannelVersion` separately as parameters, can we just pass the `ClusterVersion` object? We should be able to calculate the above from the `ClusterVersion` (moving that logic into this helper).

Reply:
Done.